.travis.yml

```diff
@@ -36,6 +36,7 @@ env:
 
    - DESTINATION="OS=10.0,name=iPhone 6s" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="YES" BUILD_EXAMPLE="YES" RUN_UI_TESTS="NO" POD_LINT="NO"
    - DESTINATION="OS=10.1,name=iPhone 6s" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="YES" BUILD_EXAMPLE="NO"  RUN_UI_TESTS="YES" POD_LINT="NO"
+   - DESTINATION="OS=10.2,name=iPhone 6s" SDK="$IOS_SDK" SCHEME="$IOS_SCHEME" RUN_TESTS="YES" BUILD_EXAMPLE="NO"  RUN_UI_TESTS="NO" POD_LINT="NO"
```

Comment:
ok, for some reason it looks like travis thinks `10.2, iPhone 6s` doesn't exist 😅 (it does)

let's try using `name=iPhone 7` for 10.2